### PR TITLE
New header logo

### PR DESCRIPTION
**Pull request by DennisTsiang**

### CODE REVIEW

File: source/assets/stylesheets/styles/_site_brand.scss [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
   @include breakpoint($tablet-breakpoint) {
-    float: left;
-    padding: 16px 20px;
+    padding-right: 20px;
   }
-}
+
+  @include breakpoint($desktop-breakpoint) {
+    padding-right: 0;
+  }}
```
Comment (neilvanbeinum) [inline source/assets/stylesheets/styles/_site_brand.scss]: Think we want to lose the extra `}` here

Comment (DennisTsiang) [reply source/assets/stylesheets/styles/_site_brand.scss]: Good spot, thanks Neil. That brace should be on a new line

File: source/assets/stylesheets/styles/_page_content.scss [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
-    margin-top: 64px;
+    margin-top: 100px;
   }
+
 }
 
 .page-content {
   @extend %page-content-margins;
   background-color: $light-gray;
+
```
Comment (neilvanbeinum) [inline source/assets/stylesheets/styles/_page_content.scss]: There's a couple of extra blank lines we should probably remove here for consistency

**Comment by neilvanbeinum** [comment]
Looks good 👍

**Pull request merged**